packages/shared/src/contexts/SquadsContext.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode, ReactElement, useMemo } from 'react';
+import { Squads } from '../graphql/squads';
+
+export const SQUADS_DEFAULTS: Squads = {
+  squads: [],
+};
+
+export interface SquadsContextData {
+  squads: Squads;
+  loadedSquads?: boolean;
+}
+
+export const MAX_DATE = new Date(3021, 0, 1);
+
+const SquadsContext = React.createContext<SquadsContextData>({
+  squads: SQUADS_DEFAULTS,
+  loadedSquads: false,
+});
+
+interface SquadsContextProviderProps {
+  children: ReactNode;
+  squads: Squads;
+  loadedSquads?: boolean;
+}
+
+export const SquadsContextProvider = ({
```

Comment:
Do we need another context for that or can we utilize an existing context?

Reply:
I didnt like adding another context but non of the current ones seemed related

Reply:
Perhaps we can use the user context?

Reply:
Done 👍